tests/tests_reload.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Ensure that crypto_policy_reload variable works
+  hosts: all
+  tasks:
+    - name: Get SSHD pid before policy update
+      shell: |
+        echo "SSHD_PID=\"$(pidof /usr/sbin/sshd)\""
+      register: sshd_pid_before
+    - name: Change policy from DEFAULT TO LEGACY (no reload)
+      include_role:
+        name: linux-system-roles.crypto_policies
+      vars:
+        crypto_policies_policy: LEGACY
+        crypto_policies_reload: false
+    - name: Get sshd pid after first update
+      shell: |
+        echo "SSHD_PID=\"$(pidof /usr/sbin/sshd)\""
+      register: sshd_pid_after_first
+    - name: Verify that policy was changed to LEGACY
+      assert:
+        that:
+          - "crypto_policies_active == 'LEGACY'"
+    - name: Check the sshd was not reloaded
+      assert:
+        that:
+          - sshd_pid_before.stdout == sshd_pid_after_first.stdout
+    - name: Change policy from LEGACY to DEFAULT (reload)
+      include_role:
+        name: linux-system-roles.crypto_policies
+      vars:
+        crypto_policies_policy: DEFAULT
+        crypto_policies_reload: true
```

Comment:
This is default so I think it should not be explicitly here. If you remove it, you also test the default value is still true.

Reply:
Good point, fixed.